api/src/main/java/org/cardanofoundation/rosetta/api/construction/service/ConstructionApiServiceImpl.java

```diff
@@ -243,15 +242,6 @@ public TransactionIdentifierResponse constructionSubmitService(
     return new TransactionIdentifierResponse(new TransactionIdentifier(txHash), null);
   }
 
-  private void checkOperationsHaveIdentifier(List<Operation> operations) {
```

Comment:
I delete this code since this issue
https://sonarcloud.io/project/issues?resolved=false&severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&sinceLeakPeriod=true&types=BUG&id=cardano-foundation_cardano-rosetta-java&open=AY-CT-Y_r9JkjpuTDa1i

Also, TS implementation doesn't have this check either. From the `ConstructionApi` we get sanitized input and that check is redundant.